src/MoBi.Assets/AppConstants.cs

```diff
@@ -495,9 +496,14 @@ public static string RemoveValuePointDescription(double xDisplayValue, double yD
             return string.Format("Remove {3} X: '{0} {5}' Y: '{1} {6}' from {4} '{2}'", xDisplayValue, yDisplayValue, formulaName, ObjectTypes.ValuePoint, ObjectTypes.TableFormula, xDisplayUnit.Name, yDisplayUnit.Name);
          }
 
-         public static string CommitCommandDescription(Module module, IModelCoreSimulation simulation)
+         public static string CommitCommandDescription(IModelCoreSimulation simulation)
          {
-            return $"Commit changes made in Simulation: '{simulation.Name}' to module: '{module.Name}'";
+            return $"Commit changes made in Simulation: '{simulation.Name}'";
```

Comment:
Here's an example where I expand the top level. Maybe it makes sense to put the module name anyway at the top one because that's the primary task of the commit.

Or do you think some other way of describing it?

![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/261477/12aef2ce-b73c-4fb2-8a3f-0968359878bd)